html/rendering/replaced-elements/attributes-for-embedded-content-and-images/img-aspect-ratio.html

```diff
@@ -53,6 +53,6 @@
   assert_ratio(images[1], 2.0); // 2.0 is the original aspect ratio of green.png
   assert_equals(getComputedStyle(images[2]).height, "0px"); // aspect-ratio doesn't override intrinsic size of images that don't have any src.
   assert_equals(getComputedStyle(images[3]).height, getComputedStyle(images[4]).height); // aspect-ratio doesn't override intrinsic size of error images.
-  assert_ratio(images[5], 1.266); // 1.266 is the original aspect ratio of blue.png
+  assert_ratio(images[5], 1.255); // 133/106 is the original aspect ratio of blue.png
```

Comment:
I agree with @fred-wang that we should just write the division (133 / 106) in the second argument.

Reply:
Yes, I meant changing to 133/106 in the argument. For the comment, I don't really mind but I guess we don't even need to repeat it e.g. "// This is the original aspect ratio of blue.png"

Reply:
Ah, sorry! I misunderstood this...